{"name": "test_run_invalid_api_wrapper", "status": "passed", "description": "Test that _run fails gracefully when `api_wrapper.run` is not implemented correctly.", "start": 1743505056544, "stop": 1743505056545, "uuid": "64f80a82-53f3-422e-9395-6ae1807bfb20", "historyId": "0cfd6ac5aa753ad68889f8c0c1757507", "testCaseId": "0cfd6ac5aa753ad68889f8c0c1757507", "fullName": "src.tests.test_base_tool.TestBaseAction#test_run_invalid_api_wrapper", "labels": [{"name": "tag", "value": "negative"}, {"name": "tag", "value": "base"}, {"name": "tag", "value": "unit"}, {"name": "parentSuite", "value": "src.tests"}, {"name": "suite", "value": "test_base_tool"}, {"name": "subSuite", "value": "TestBaseAction"}, {"name": "host", "value": "fv-az1673-369"}, {"name": "thread", "value": "2350-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.tests.test_base_tool"}]}